.taskcat.yml

```diff
@@ -2,67 +2,22 @@ project:
   name: cfn-ecr-aws-soci-index-builder
   owner: quickstart@amazon.com
   s3_regional_buckets: true
+  s3_bucket: soci-index-builder
```

Comment:
This will need to be configurable since s3 bucket names need to be globally unique.

Reply:
Setting the S3 bucket name to `soci-index-builder` allowed for creating the Cloudformation stack without needing to input an explicit bucket name. Since, we cannot have a fixed bucket name due to global uniqueness, I am fallinbg back to using `[taskcat_autobucket]`  which will assign a taskcat specific bucket name `tcat-12345..`. Downside is that now when we create the cloudformation stack from the provided template, we will need to specify the exact taskcat bucket name. 

I have opened an issue in taskcat https://github.com/aws-ia/taskcat/issues/856 regarding adding a new psudo parameter, such that we can use `AWS::AccountId` in the cloudformation template to help choose the bucket automatically.